Bottom: 4a7dab2b01b7aaaed4b23c5e3e3713745e6e7124
Top:    8b615ed183c15b7cbb0a58b9a6872be5e52cefc6
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-11-21 21:05:50 +0000

Refresh of play-with-glsl-shader-for-rend

---

diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index e5cbc4a..36727d0 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -1806,9 +1806,6 @@ ghid_drawing_area_expose_cb (GtkWidget *widget,
   int new_x, new_y;
   int min_depth;
   int max_depth;
-  static float wavetime = 0;
-  extern GLuint sp;
-  GLint waveTimeLoc = glGetUniformLocation (sp, "waveTime");
 
   buffer.total_triangles = 0;
   buffer.total_vertices = 0;
@@ -1819,9 +1816,6 @@ ghid_drawing_area_expose_cb (GtkWidget *widget,
   hidgl_init ();
   check_gl_drawing_ok_hack = true;
 
-  wavetime += 0.1;
-  glUniform1f (waveTimeLoc, wavetime);
-
   /* If we don't have any stencil bits available,
      we can't use the hidgl polygon drawing routine */
   /* TODO: We could use the GLU tessellator though */
